tests/bud.bats

```diff
@@ -262,3 +262,16 @@ load helpers
   [ "$status" -eq 0 ]
   [ "$output" = "" ]
 }
+
+@test "bud-shell" {
+  target=alpine-image
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/shell
+  run buildah --debug=false inspect --type=image --format '{{.Docker.Config.Shell}}' ${target}
+  echo $output
+  [ "$status" -eq 0 ]
+  [ "$output" = "[/bin/sh -c]" ]
```

Comment:
This test should probably also verify that `buildah config --shell` updates the value.